examples/api/lib/material/expansion_tile/expansion_tile.1.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for [ExpansionTile].
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ExpansionTileApp());
+
+class ExpansionTileApp extends StatelessWidget {
+  const ExpansionTileApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(useMaterial3: true, colorSchemeSeed: const Color(0xff6750a4)),
```

Comment:
The default color scheme, without specifying a colorSchemeSeed, should be OK for examples.

Reply:
This is just the same code sample but with `useMaterial3: true` since we're now enabling this in all examples, I can remove this example entirely.